Api/OneDrive/OneDriveProvider.cs

```diff
@@ -240,12 +242,5 @@ public override string Email
 				return email;
 			}
 		}
-
-		public override async Task Logout()
-		{
-			Api.ResetData();
-			await RemoveApi(Id);
-			ApiManager.Shared.SaveApi(Api);
```

Comment:
//cc @Clancey 

Why is this provider saving the API when logging out? It looks like it's a mistake, but i thought i should point it out explicitly as i did remove this call during the refactor